edaR/R/printSingleVarStats.R

```diff
@@ -6,6 +6,10 @@
 #' @export
 #'
 printSingleVarStats <- function(single_var_stat_object){
+  #set pander to false auto.asis
+  old_panderOption <- panderOptions('knitr.auto.asis')
+  panderOptions('knitr.auto.asis', FALSE)
+  
   cat('## Single Var Interactions \n')
```

Comment:
You can also use some other helper functions from the `pander` package, eg here:

```r
> pandoc.header('Single Var Interactions', level = 2)

## Single Var Interactions
```